Formula/catch2.rb

```diff
@@ -0,0 +1,32 @@
+class Catch2 < Formula
+  desc "Modern, C++-native, header-only, test framework"
+  homepage "https://github.com/catchorg/Catch2"
+  url "https://github.com/catchorg/Catch2/archive/v2.5.0.tar.gz"
+  sha256 "720c84d18f4dc9eb23379941df2054e7bcd5ff9c215e4d620f8533a130d128ae"
+
+  depends_on "cmake" => :build
+
+  def install
+    inreplace "CMakeLists.txt", "{CMAKE_INSTALL_LIBDIR}/cmake", "{CMAKE_INSTALL_DATAROOTDIR}/cmake"
```

Comment:
Why is this necessary? Has it been reported upstream?

Reply:
No, this is not really necessary. It works out of box so nothing to report. But it feels better if we use the default path.